contrib/vagrant/start.sh

```diff
@@ -437,6 +437,9 @@ function set_vagrant_env(){
         echo "# iptables -I INPUT -p tcp -s ${IPV4_BASE_ADDR_NFS}0/24 --dport 111 -j ACCEPT"
         echo "# iptables -I INPUT -p tcp -s ${IPV4_BASE_ADDR_NFS}0/24 --dport 2049 -j ACCEPT"
         echo "# iptables -I INPUT -p tcp -s ${IPV4_BASE_ADDR_NFS}0/24 --dport 20048 -j ACCEPT"
+
+        echo "# You might also want to add the following route to reach the control node:"
+        echo "# ip route add $MASTER_IPV4 via $MASTER_IPV4_NFS"
```

Comment:
When is this needed? I didn't add such a route on my system :/

Reply:
Was based on a suggestion from @aanm to use `kubectl` from the host since `$MASTER_IPV4` is not accessible from host.

Ref: https://cilium.slack.com/archives/C2B917YHE/p1596096556407100

![image](https://user-images.githubusercontent.com/221481/95331329-21d4f880-08aa-11eb-9eda-c1cd9d2b6968.png)

Reply:
please add an explanation why someone would need to add that route. I usually access `kubectl` from the VMs.

Reply:
> please add an explanation why someone would need to add that route. I usually access `kubectl` from the VMs.

The current text is:
```
# You might also want to add the following route to reach the control node:
```

Is not that clear enough? Suggestions to improve it?

Reply:
Maybe:

> You also need to add the following route to use kubectl on the host.

Reply:
SGTM!